Description: Remove obsolete systemd targets
 syslog.target is now socket-activated and hence obsolete
Author: Felix Lechner <felix.lechner@lease-up.com>
Forwarded: no
Last-Update: 2017-12-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/svxlink/systemd/remotetrx.service.in
+++ b/src/svxlink/systemd/remotetrx.service.in
@@ -3,7 +3,7 @@
 [Unit]
 Description=SvxLink remote transceiver repeater control software
 Requires=svxlink_gpio_setup.service
-After=network.target remote-fs.target syslog.target time.target
+After=network.target remote-fs.target time.target
 After=svxlink_gpio_setup.service
 
 [Service]
--- a/src/svxlink/systemd/svxlink.service.in
+++ b/src/svxlink/systemd/svxlink.service.in
@@ -3,7 +3,7 @@
 [Unit]
 Description=SvxLink repeater control software
 Requires=svxlink_gpio_setup.service
-After=network.target remote-fs.target syslog.target time.target
+After=network.target remote-fs.target time.target
 After=svxlink_gpio_setup.service
 
 [Service]
